Description: Build with qtwebkit instead of qwbengine
 Qwebengine bundles parts of the chromium web browser and might not be
 packaged for Debian any time soon. This patch makes Calamares use qtwebkit
 instead.
 .
 calamares (2.2.2-1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: Jonathan Carter <jcarter@linux.com>

---

Origin: other
Last-Update: 2016-05-16

Index: calamares-2.4.5/CMakeLists.txt
===================================================================
--- calamares-2.4.5.orig/CMakeLists.txt	2016-11-25 19:38:44.000000000 +0200
+++ calamares-2.4.5/CMakeLists.txt	2016-12-07 13:54:10.000000000 +0200
@@ -57,6 +57,7 @@
 
 option( WITH_PYTHON "Enable Python modules support." ON )
 option( WITH_CRASHREPORTER "Build with CrashReporter" ON )
+option ( WEBVIEW_FORCE_WEBKIT "Build with QtWebKit instead of QWebEngine" ON )
 
 if( CMAKE_SYSTEM_PROCESSOR MATCHES "arm" OR NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/libcrashreporter-qt/CMakeLists.txt" )
     message( STATUS "Build of crashreporter disabled." )
